main.lua

```diff
@@ -643,27 +649,29 @@ function update_audio(buffer)
 	end
 end
 
-local function isCtrlOrGuiDown()
-	return (love.keyboard.isDown('lctrl') or love.keyboard.isDown('lgui') or love.keyboard.isDown('rctrl') or love.keyboard.isDown('rgui'))
-end
-
 function love.keypressed(key)
-	if cart and pico8.cart._keydown then
-		return pico8.cart._keydown(key)
-	end
+
 	if key=='r' and isCtrlOrGuiDown() then
 		_load()
+
```

Comment:
Do these blank lines serve much of a purpose? The code is readable to me, at least, without them. (Not too hung up about it though.)

Reply:
Sorry, they only mark where I’m working and more stuff will be added later or they’ll be Removed when I’m done